docs/src/resample_proj.md

```diff
@@ -0,0 +1,124 @@
+## resample and Projections with ProjString
+
+Geospatial datasets will come in different [projections](https://proj.org/en/9.4/operations/projections/index.html) for whaever reasons. Here, we will focus in some of the most used ones, e.g., `MODIS SINUSOIDAL` and `EPSG`, as well as transformations between them.
+
+Let's start by loading the neccesary packages:
+
+````@example modis
+using Rasters, RasterDataSources, ArchGDAL
+using DimensionalData
+using DimensionalData.Lookups
+using NaNStatistics
+using CairoMakie
+````
+
+and let's load a test raster
+
+````@example modis
+ras = Raster(WorldClim{BioClim}, 5)
+ras_m = replace_missing(ras, missingval=NaN)
+````
+
+and let's also take a look
+
+````@example modis
+fig = plot(ras_m; colorrange=(0,100))
+````
+
+### MODIS SINUSOIDAL PROJECTION
+
+````@example
+# ? is this the right ProjString ?, do we need to shift lat, lon?
+SINUSOIDAL_CRS = ProjString("+proj=sinu +lon_0=0 +x_0=0 +y_0=0 +a=6371007.181 +b=6371007.181 +units=m +no_defs")
```

Comment:
Users will be wondering what the hell a ProjString is, we should explain that

Reply:
also @lazarusA why are you defining the ellipse here?  That seems like a step too far.  How about:

```julia
SINUSOIDAL_CRS = ProjString("+proj=sinu +lon_0=0 +type=crs")

Reply:
Those tiles come/are set with very specific numbers. I think I did some tests, and you not always get the same numbers. BTW, feel free to merge this into your PR, I will probably not finish this anytime soon.